.github/workflows/php-tests-wp-nightly-php-7-4.yml

```diff
@@ -32,6 +51,7 @@ jobs:
       MYSQL_ROOT_PASSWORD: root
       WP_VERSION: nightly
     steps:
+      - uses: styfle/cancel-workflow-action@0.8.0
       - uses: actions/checkout@v2
       - name: Grant database access
```

Comment:
Looks like this PR needs to be synced with develop since this step is no longer here.